backend/drtrottoir/models.py

```diff
@@ -96,6 +96,10 @@ class ScheduleDefinitionBuilding(models.Model):
     )
     position = models.IntegerField()
 
+    # class Meta:
```

Comment:
Remove commented code, if this isn't required anymore.

Reply:
I'd like to keep this here, as I will be implementing this in a following PR and need a reference.